nasa/openmct
# [MCT3739] Updating document title when domainObject name mutates

Closes #3739

A listener was added to observe the mutation on the domainObject that is selected. If the `name` property is updated by either the `BrowseBar` or the `PropertiesActions`, the listener will update the document title.

### How to Test
Navigate to a domain object in the file tree and select it. 
* Update the name through the browse bar, observe it has updated the document title. 
* Right click on the selected domain object in the tree, click Edit Properties. Update the name and save the dialog. Observe that the changed name has been reflected in the document title.
* Ensure that changing a domain object that is not selected, does not update the document title.

### All Submissions:


### Author Checklist
